### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by geojaz**
I originally was going to remove the every other odd week typo. But the scope of this expanded...  a little bit. O:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,33 +179,33 @@ https://github.com/kubernetes/test-infra/blob/master/commands.md).
...
 
-This time is focused on developers, although we will never turn a courteous participant away. Please swing by, even if you've never actually installed kops.
+Office hours are designed for ALL of those contributing to kops or the community. Contributions are not limited to commiting source code. There are so many important ways to be involved- 
+ - helping in the slack channels
+ - triaging/writing issues 
+ - thinking about the topics raised at office hours and forming and advocating for your good ideas  forming opinions
+ - testing pre-(and official) releases
 
-We encourage you to reach out **beforehand** if you plan on attending. You're welcome to join any session, and please feel free to add an item to the  [agenda](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit) where we track notes from office hours.
+Although not exhaustive, the above activityies are extremely important to our continued success and are all worth contributions. If you want to talk about kops and you have doubt, just come.
```
Comment (chrisz100) [inline README.md]: Activities - typo ;-)

**Comment by chrisz100** [comment]
/lgtm 

;-)

File: README.md [hunk 2]
```diff
@@ -179,33 +179,33 @@ https://github.com/kubernetes/test-infra/blob/master/commands.md).
...
+We do maintain an [agenda](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit) and stick to it as much as possible. If you want to hold the floor, put your item in this doc. Bullet/note form is fine. Even if your topic gets in late, we do our best to cover it.
 
-- Help and guide to those who attend, who are interested in contributing.
-- Discuss the current state of the kops project, including releases.
-- Strategize about how to move `kops` forward.
-- Collaborate about open and upcoming PRs.
-- Present demos.
+Our office hours call is recorded, but the tone tends to be casual. First-timers are always welcome. Typical areas of discussion can include:
+- Contributors with a feature proposal seeking feedback, assistance, etc
+- Memembers planning for what we want to get done for the next release
```
Comment (mikesplain) [inline README.md]: Typo members

**Comment by mikesplain** [comment]
/hold cancel

Haha sorry for all that, working from iPad at the moment!

**Comment by mikesplain** [comment]
Awesome thanks!

/lgtm
/approve

**Pull request merged**